custom.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/grupozap/aegir/internal/pkg/utils"
+	livr "github.com/k33nice/go-livr"
+)
+
+func neq(args ...interface{}) livr.Validation {
+	not_allowed := utils.FirstArg(args...)
+
+	return func(value interface{}, builders ...interface{}) (interface{}, interface{}) {
+		if value == nil || value == "" {
+			return nil, nil
+		}
+
+		switch value.(type) {
+		case float64, string, bool:
+		default:
+			return nil, errors.New("FORMAT_ERROR")
+		}
+		if fmt.Sprint(value) == fmt.Sprint(not_allowed) {
+			return nil, errors.New("NOT_ALLOWED_VALUE")
+		}
+
+		return value, nil
+	}
+}
+
+func not_like(args ...interface{}) livr.Validation {
+	var re *regexp.Regexp
+	var flags string
+	if len(args) > 0 {
```

Comment:
I have a little suggestion (I didn't test, I don't know if it will work correctly :sorry:) 

```suggestion
	if len(args) > 0 {
		if v, ok := args[0].(string); ok {
			re, err := regexp.Compile(flags + v)
			
			if err != nil {
				re = regexp.MustCompile(".*")
			}
		}
	}

	if len(args) > 1 {
		if v, _ := args[1].(string); v == "i" {
			flags = "(?i)"
		}
	}
```

Reply:
I agree with all suggestions but I won't change this one, because for me, it's more concise but yours is more explicit. And I don't think it will change much the result or readability, but I appreciate it anyway.

Thank you for the awesome review! <3